components/recent-header.js

```diff
@@ -1,10 +1,36 @@
 import { ITEM_TYPES, ITEM_TYPES_UNIVERSAL } from '@/lib/constants'
+import BootstrapForm from 'react-bootstrap/Form'
 import { Select } from './form'
 import { useRouter } from 'next/router'
 
+function mapTypeToSelectOption (type) {
+  switch (type) {
+    case 'bounties_active': return 'bounties'
+  }
+
+  return type
+}
+
+function ActiveBountiesCheckbox ({ router, prefix }) {
+  const onChange = (e) => {
+    const route = e.target.checked ? 'bounties_active' : 'bounties'
+
+    router.push(prefix + `/recent/${route}`)
+  }
+
+  return (
+    <div className='mx-2 my-1'>
```

Comment:
Checkbox isn't properly aligned with the dropdown:

![2024-11-17-232405_1920x1080_scrot](https://github.com/user-attachments/assets/66ff5146-c982-4ab9-a7ad-df1c9cd3a0b2)
![2024-11-17-232401_1920x1080_scrot](https://github.com/user-attachments/assets/5bf327f6-d59f-4b9c-914f-53ddc3176252)

This fixes it:

```suggestion
    <div className='mx-2 mb-2'>
```